themes/coderdojoschoeneweide/layouts/_default/baseof.html

```diff
@@ -3,6 +3,31 @@
 <html lang="{{.Site.LanguageCode}}" dir="{{.Site.Language.LanguageDirection}}">
 {{ block "head.html" . }}
 <head>
+
+    <div class="abod-lang-2">
```

Comment:
Der ganze Block sollte wahrscheinlich in den \<header\>-Tag (weiter unten) anstatt in den \<head\>-Tag, oder? :D